README.md

```diff
@@ -0,0 +1,27 @@
+# pl-slideshow
+An example plugin to demonstrate CHRiS' parallelization capabilities and to provide a framework for building your own parallelized CHRiS plugin
```

Comment:
Maybe make a statement somewhere up front that this repo is both a framework and an example of using the framework.  And that if you want to build your own parallelized plugin, your approach should be ...